docs/release-notes/2.40.1.md

```diff
@@ -0,0 +1,20 @@
+# Percona Monitoring and Management 2.40.1
+
+| **Release date:** | Oct 18, 2023                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open-source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+## Fixed issues
+
+- [PMM-12592](https://jira.percona.com/browse/PMM-12592) - Fixed an issue where the **Library Panels** for the PMM dashboard were not working.
+- [PMM-12576](https://jira.percona.com/browse/PMM-12576) - After upgrading to PMM version 2.40.0, changing the Admin user's password from the terminal was not functioning. The issue has been resolved now. This impacts all versions prior to PMM 2.40.0.
+- [PMM-12587](https://jira.percona.com/browse/PMM-12587) - During the upgrade from version 2.39.0 to 2.40.0, the process failed with `500 internal server error` and an indefinite loop of `pmm-update-perform-init service` because of incorrect mappings between dashboards, folders, users, and groups. The issue has been resolved.
```

Comment:
1) indefinite loop of `pmm-update-perform-init service` - as s User I can't see this from UI, so this is redundant.
2) Was it always failing? As far as afaik - it can end up updating BUT!  the folders, users and their permissions are broken/mixed 

@BupycHuk PTAL

Reply:
sometimes it end-up with indefinite loop. I think we shouldn't mention 2.39 here, because upgrade could be done from any version prior 2.40.0

Reply:
ok.